UnitTests/CardTests/Card_Tests.swift

```diff
@@ -0,0 +1,15 @@
+import XCTest
+@testable import Card
+
+final class Card_Tests: XCTestCase {
+    func testCard_setsProperExpiryStringFormat() {
```

Comment:
Fixes issue where this file wasn't actually compiling / running in the previous test suite.